src/server.js

```diff
@@ -157,7 +157,7 @@ const main = async () => {
 
   /**
    * @api {post} /api/syncProfile Update mixpanel profile
-   * @apiDescription Syncs profile info from orchestration to mixpanel
+   * @apiDescription Syncs profile info from orchestration to mixpanel. Requires an authorization token to be set pull the profile from Orchestration.
```

Comment:
I might be misreading this but I'm having a little trouble understanding this sentence in what is happening with Orchestration. Is the token pulling the profile from Orch?

Reply:
That's a good question. From what I can tell, it is pulling the profile associated with a token, and syncing it into mixpanel. I don't know where we would use this though.

Reply:
I will try a different wording.